### ISSUE TITLE
What modifications should I make to make it work on my Korean client as well?

### ISSUE THREAD

**Issue by chunhyowoon** [Barragek0/ClickIt#17]
If I try to make them recognize Korean, which part should I revise?
clickitSettings.cs 의
[Menu("Map Boss: Gain (50 to 80)% of Physical Damage as Extra Damage of a random Element\n" +
" Damage Penetrates (15 to 25)% of Enemy Elemental Resistances", 15, 5000)]
Can I edit these parts of the code above into Korean?
Please tell me where to change so that I can recognize Korean

**Comment by Barragek0** [comment]
Yes, changing every string in clickitsettings to the korean one that you see in-game should allow it to work. Though it might be a bit more complex and you might need to do more messing around to get it to work.  Chatgpt should be able to help you do this.
